profiler/ann_data.py

```diff
@@ -0,0 +1,23 @@
+from time import perf_counter
+
+import cellxgene_census
+
+import tiledbsoma as soma
+
+census_S3_latest = dict(census_version="latest")
+census_local_copy = dict(uri="/Users/brobatmili/projects/census_data/")
+
+
+def main():
+    t1 = perf_counter()
+    with cellxgene_census.open_soma(**census_local_copy) as census:
```

Comment:
I don't think we should use Census API here. We can do the equivalent with just TileDBSOMA API.